tests/network/bindingplugin_passt.go

```diff
@@ -109,13 +109,13 @@ var _ = Describe(SIG(" VirtualMachineInstance with passt network binding plugin"
 		Expect(console.RunCommand(vmi, cmd, time.Second*5)).To(Succeed())
 	})
 
-	Context("TCP without port specification", func() {
+	Context("TCP without port specification", Ordered, decorators.OncePerOrderedCleanup, func() {
 		var clientVMI *v1.VirtualMachineInstance
 		var serverVMI *v1.VirtualMachineInstance
 
 		const highTCPPort = 8080
 
-		BeforeEach(func() {
+		BeforeAll(func() {
```

Comment:
I don't understand why did you use `BeforeAll()` instead of `BeforeEach(OncePerOrdered, )` like in the above examples.

The [docs](https://onsi.github.io/ginkgo/#setup-around-ordered-containers-the-onceperordered-decorator) say:
```
we retain the existing behavior for the entire suite and get the BeforeAll-like behavior we need for our Ordered container.
```

What am I missing?

Reply:
The docs used it for special cases where the fixtures are used for both scenarios, i.e. for ordered and un-ordered tests.

For clarity and better validation (`BeforeAll` is not allowed to be defined in a non-ordered block), the recommendation is to use `BeforeAll`.